README.md

```diff
@@ -26,21 +26,47 @@ To install the package from source and use the latest release, you can clone the
 pip install git+https://github.com/julien-hec/pyBKTR.git
 ```
 
+### Notes
+If you obtain an error message when installing the package, it may be due to the installation of the `torch` package.
+The `torch` package is a dependency of the `pyBKTR` package and there is a good chance that the error comes from the installation of `torch`.
```

Comment:
```suggestion
The `torch` package is a dependency for the `pyBKTR` package and there is a good chance that it could be responsible for encountered installation errors.
```